### PR TITLE
Update store-apps-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as the Microsoft store interface has changed

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2952

**Comment by PRMerger8** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/apps/store-apps-windows.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ You can add a Microsoft Store app to Intune by doing the following:
 2. Select **Apps** > **All apps** > **Add**.
 3. In the **Select app type** pane, under the available **Store app** types, select **Microsoft store app**.
 4. Click **Select**. The **Add app** steps are displayed.
-5. To configure the **App information** for Microsoft store apps, navigate to the [Microsoft store](https://www.microsoft.com/store/apps) and search for the app you want to deploy. Display the app page and make a note of the app details. 
+5. To configure the **App information** for Microsoft store apps, navigate to the [Microsoft store](https://www.microsoft.com/store/apps), Click on **Open Store Apps** and search for the app you want to deploy. Display the app page and make a note of the app details.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/store-apps-windows.md:47]:
```suggestion
5. To configure the **App information** for Microsoft store apps, navigate to the [Microsoft store](https://www.microsoft.com/store/apps), select **Open Store Apps**, and search for the app you want to deploy. Display the app page and make a note of the app details. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger4** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and Sign Off this PR. Thanks

**Comment by yogkumgit** [comment]
@Erikre Gentle reminder

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**